ReactWindows/ReactNative/Views/Image/ReactImageManager.cs

```diff
@@ -17,14 +23,17 @@ namespace ReactNative.Views.Image
     /// <summary>
     /// The view manager responsible for rendering native images.
     /// </summary>
-    public class ReactImageManager : SimpleViewManager<Border>
+    public class ReactImageManager : BaseViewManager<Border, ReactImageShadowNode>
     {
         private readonly Dictionary<int, SerialDisposable> _disposables =
             new Dictionary<int, SerialDisposable>();
 
         private readonly Dictionary<int, List<KeyValuePair<string, double>>> _imageSources =
             new Dictionary<int, List<KeyValuePair<string, double>>>();
 
+        private Color? _tintColor;
```

Comment:
>Color [](start = 16, length = 5)

The view manager instance is a singleton in the application, meaning there is a one to many relationship between the view manager and the views it instantiates.  Thus, you can't really assign properties to the view manager.  The correct way to do this is one of two ways, either create a dictionary like the ones above to associate tint color to React view tag, or to collect all the relevant properties in the shadow node and pass them through via the UpdateExtraData API.